### DISCUSSION TITLE
Available parsers for IOS

### DISCUSSION THREAD

**Discussion by chet-space** [napalm-automation/napalm-logs#361] [Q&A] [answered]
Hey all,

I have napalm-logs setup and I'm curious if there are additional parsers available for syslog messages. I pointed the config_path to napalm-logs/config/ but there are only a few parsers in that directory. Are there more available that I'm missing?

**Comment by mirceaulinic** [comment, marked as answer]
Hey @chet-space. By default, napalm-logs will load these parsers for IOS: https://github.com/napalm-automation/napalm-logs/tree/develop/napalm_logs/config/ios.
If you need more (and you probably do), you can provide them into your own environment, and point napalm-logs to load them from there, using the `extension_config_path` configuration option, see https://github.com/napalm-automation/napalm-logs/blob/develop/docs/options/index.rst#extension-config-path.
Ideally, whenever the parser is sufficiently generic, I would encourage you to open a PR to add it here, to be part of the library.
